sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/crypto/_client.py

```diff
@@ -126,14 +127,24 @@ def key_id(self):
         """
         return self._key_id.source_id
 
+    @classmethod
+    def from_jkw(cls, key_id, jwk):
+        # type: (str, dict) -> CryptographyClient
+        """Creates a client that can only perform cryptographic operations locally.
+
+        :param str key_id: the full identifier of an Azure Key Vault key with a version.
+        :param dict jwk: the key's cryptographic material, as a dictionary.
+        """
+        return cls(KeyVaultKey(key_id, jwk), TokenCredential(), local_only=True)
+
     @distributed_trace
     def _initialize(self, **kwargs):
         # type: (**Any) -> None
         if self._initialized:
             return
 
         # try to get the key material, if we don't have it and aren't forbidden to do so
-        if not (self._key or self._keys_get_forbidden):
+        if not (self._key or self._keys_get_forbidden or self._local_only):
```

Comment:
Since key material is required to create a local-only client, this is theoretically redundant since self._key shouldn't be None. A user _could_ set self._key = None though, so I added this check just in case. It doesn't always make sense to account for the various ways users could interfere with the class though, so are "just in case" checks like this unnecessary entirely?

Reply:
Yes. An underscored attribute is private, makes no compatibility promises.